src/test/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/beregning/TilsynBarnVedtaksperiodeValidingerUtilsTest.kt

```diff
@@ -299,31 +317,33 @@ class TilsynBarnVedtaksperiodeValidingerUtilsTest {
 
         @Test
         fun `skal godta stønadsperiode på tvers av 2 godkjente sammenhengende vilkårsperioder`() {
-            val vedtaksperiode = lagVedtaksperiode(fom = LocalDate.of(2023, 1, 1), tom = LocalDate.of(2023, 1, 10))
+            val vedtaksperiode = lagVedtaksperiode(fom = LocalDate.of(2025, 1, 1), tom = LocalDate.of(2025, 1, 10))
 
             assertThatCode {
                 tilsynBarnVedtaksperiodeValidingerService.validerVedtaksperioder(
                     listOf(
                         vedtaksperiode,
                     ),
                     behandlingId,
+                    utgifter,
                 )
             }.doesNotThrowAnyException()
         }
 
         @Test
         fun `skal ikke godta stønadsperiode på tvers av 2 godkjente, men ikke sammenhengende vilkårsperioder`() {
-            val vedtaksperiode = lagVedtaksperiode(fom = LocalDate.of(2023, 1, 1), tom = LocalDate.of(2023, 1, 21))
+            val vedtaksperiode = lagVedtaksperiode(fom = LocalDate.of(2025, 1, 1), tom = LocalDate.of(2025, 1, 21))
 
             assertThatCode {
                 tilsynBarnVedtaksperiodeValidingerService.validerVedtaksperioder(
                     listOf(
                         vedtaksperiode,
                     ),
                     behandlingId,
+                    utgifter,
                 )
             }.hasMessageContaining(
-                "Finnes ingen periode med oppfylte vilkår for AAP i perioden 01.01.2023 - 21.01.2023",
```

Comment:
Hvorfor må datoene i testene oppdateres? Er det noe med testoppsettet som gjør at de begynte å feile?

Reply:
Ja, det kunne jeg ha beskrevet bedre. Nå må det være utgifter hele vedtaksperioden. Default utgifter i testene er for januar og februar 2025, så har flyttet alle testene til å være rundt denne perioden.